2011-04-11  Joseph Myers  <joseph@codesourcery.com>

	* ports/sysdeps/mips/dl-tls.h (TLS_DTV_UNALLOCATED): Define.

diff --git a/ports/sysdeps/mips/dl-tls.h b/ports/sysdeps/mips/dl-tls.h
index 6d3ed6f..75facb0 100644
--- a/ports/sysdeps/mips/dl-tls.h
+++ b/ports/sysdeps/mips/dl-tls.h
@@ -44,3 +44,6 @@ extern void *__tls_get_addr (tls_index *ti);
 
 # define GET_ADDR_OFFSET	(ti->ti_offset + TLS_DTV_OFFSET)
 # define __TLS_GET_ADDR(__ti)	(__tls_get_addr (__ti) - TLS_DTV_OFFSET)
+
+/* Value used for dtv entries for which the allocation is delayed.  */
+#define TLS_DTV_UNALLOCATED	((void *) -1l)
